### DISCUSSION TITLE
Serve own cgi scripts on lighthttpd independently from RaspAP

### DISCUSSION THREAD

**Discussion by philipan** [RaspAP/raspap-webgui#1117] [Q&A]
I try to serve own cgi-scripts for other tasks (saving data from https://sensor.community/en/sensors/airrohr/):

In /etc/lighttpd/conf-enabled/10-cgi.conf I added an additional cgi directory:

`alias.url += ( "/cgi/" => "/var/www/cgi/" )
$HTTP["url"] =~ "/cgi/" {
        cgi.assign = (
              ".pl" => "/usr/bin/perl",
              ".py" => "/usr/bin/python",
              ".cgi" => "/bin/bash",
        )
}`

However, if the scripts are called (e.g. 10.3.141.1/cgi/test.cgi), only the contents of the raspap config page is shown, however without graphics. Does anyone know how to enable an own cgi directory on the same lighthttpd server beside the raspap pages? Thanks for any help!
![raspap](https://user-images.githubusercontent.com/3703118/152740866-c69a2b38-5b39-459f-b925-ccf53c45e341.png)

**Comment by billz** [comment]
This has been asked by users in other forums, so I felt it deserved a dedicated FAQ:
https://docs.raspap.com/faq/#custompages

Hope you find this useful.
